doc/manual/import.rst

```diff
@@ -129,7 +129,7 @@ Prepare a file called ``teams.json`` which contains the teams.
 It should be a JSON array with objects, each object should contain the following
 fields:
 
-- ``id``: the (integer) team ID. Must be unique
+- ``id``: the team ID. Must be unique
```

Comment:
when is this no longer an int?

Reply:
When you import from something that is not the icpc cms it can be. Fact is the api spec says it’s a string, we just only supported ints because that is what we used. I don’t see a reason not to allow other strings

Reply:
so we are no longer using our own auto-increment ids then? In places where we display currently `t42` (as example) we display in the future `t<whateverexternalsystem decided to enter :-)>`?

Reply:
Noooo only in the api. Just like we already did with contests, problems, etc. This just unifies it